packages/model-viewer/src/test/features/animation-spec.ts

```diff
@@ -137,6 +170,39 @@ suite('ModelViewerElementBase with AnimationMixin', () => {
         });
       });
 
+      suite('when playOnce is invoked', () => {
```

Comment:
This appears to duplicate the suite you added above? Perhaps a copy/paste error?

Reply:
This is a test to see that it works correctly if you call playOnce when configured to autoplay.
But it doesn't work correctly. I will fix it.